solvency2sf/corr_h_slt.csv

```diff
@@ -0,0 +1,7 @@
+corr_h_slt,mortality,longevity,morbidity,expense,revision,lapse
+mortality,1.00,0.25,0.25,0.25,0.00,0.00
+longevity,0.25,1.00,0.00,0.25,0.25,0.25
```

Comment:
shouldn't correlation mortality vs longevity be negative, -0.25 instead of 0.25?

Reply:
good spot!

Reply:
@jeanm-m latest commit fixes this